File: app-components/src/main/java/org/synsystems/onlypass/components/logging/CrashlyticsTreeModule.java
package org.synsystems.onlypass.components.logging;

import org.synsystems.onlypass.components.AppScope;
import org.synsystems.onlypass.components.Environment;

import dagger.Module;
import dagger.Provides;

@Module
public class CrashlyticsTreeModule {
  @Provides
  @AppScope
  public CrashlyticsTree provideCrashlyticsTree(final Environment environment) {
    return CrashlyticsTree.withMinimumLevelFiltering(LogLevel.WARNING);
  }
}

File: app-components/src/main/java/org/synsystems/onlypass/components/logging/CrashlyticsTreeModule.kt
package org.synsystems.onlypass.components.logging

import dagger.Module
import dagger.Provides
import org.synsystems.onlypass.components.AppScope
import org.synsystems.onlypass.components.Environment

@Module
class CrashlyticsTreeModule {

  @Provides
  @AppScope
  fun provideCrashlyticsTree(environment: Environment) = CrashlyticsTree(LogLevel.WARNING)
}